src/ontology/uberon-edit.obo

```diff
@@ -223972,6 +223984,32 @@ is_a: UBERON:0006983 ! anatomical point
 relationship: dc-contributor https://orcid.org/0000-0001-8415-4848 ! Helena Machado
 relationship: part_of UBERON:0001697 ! orbit of skull
 
+[Term]
+id: UBERON:8500006
+name: nasal notch
+def: "Anatomical point of cranium between nasal process of premaxilla and nasal bone, at the anteriormost end of the suture." [isbn:9783830410690]
+synonym: "nasoincisve notch" EXACT []
+intersection_of: UBERON:4200230 ! surface of bone
+intersection_of: part_of UBERON:0001474 ! bone element
```

Comment:
There must be a mistake here, as I don’t see how the “nasal notch” can be defined as a “surface of bone that is part of a bone element”.

In fact the [original new term request](https://github.com/obophenotype/uberon/issues/3027) did not suggest a logical definition for “nasal notch”, just the relationships

> 'part of' some premaxilla

and

> 'part of' some 'nasal bone'

Given the text definition (“Anatomical point of cranium…”), I’d suggest removing the logical definition and add a simple `is_a: UBERON:0006983 ! anatomical point`.

Reply:
Yes, SubClassOf, as you have now done, thanks!